Source/Core/Common/GL/GLInterface/GLInterface.cpp

```diff
@@ -6,7 +6,9 @@
 
 #include "Common/GL/GLInterfaceBase.h"
 
-#if defined(__APPLE__)
+#if defined IOS
+#include "Common/GL/GLInterface/IGL.h"
```

Comment:
As discussed on IRC. You should stick with the official apple name of [EAGL](https://developer.apple.com/library/ios/documentation/OpenGLES/Reference/EAGL_functions/) instead of making up a new one.